components/todo-list.js

```diff
@@ -1,19 +1,19 @@
-import React from "react";
-import styled from "styled-components";
+import React, { useState } from 'react';
 
 import TodoItem from "./todo-item";
 
-export default function TodoList() {
+import styled from "styled-components";
+
+export default function TodoList({todos, todoRemove}) {
+
   return (
     <Wrapper>
-      Todo list가 들어가는 자리입니다!
-      <TodoItem />
+        {todos.map(todo => <TodoItem key={todo.id} {...todo} todoRemove={todoRemove}/>)}
     </Wrapper>
   );
 }
 
 const Wrapper = styled.div`
-  border: solid 1px;
   font-size: 18px;
-  flex: 1;
+  flex: 1 1 0%;
```

Comment:
```suggestion
  flex: 1;
```
```flex: 1```은 ```flex: 1 1 0%```와 똑같습니다. [참고](https://heropy.blog/2018/11/24/css-flexible-box/#flex)